src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceITTest.java

```diff
@@ -188,68 +187,6 @@ void handlePaginationError() {
-            .firstResult(0)
-            .maxResults(10)
-            .searchTaskRequest(searchTaskRequest)
-            .roleAssignments(roleAssignmentsWithGrantTypeBasic(Classification.PUBLIC))
-            .expectedAmounfOfTasksInResponse(2)
-            .expectedTotalRecords(2)
-            // taskId and caseId
-            .expectedTaskDetails(newArrayList(
-                                     "8d6cc5cf-c973-11eb-bdba-0242ac111007", "1623278362431007",
-                                     "8d6cc5cf-c973-11eb-bdba-0242ac111027", "1623278362431027"
-                                 )
-            ).build();
-
-        final TaskQueryScenario privateClassification = TaskQueryScenario.builder()
-            .scenarioName("basic_grant_type_with_classification_as_private")
-            .firstResult(0)
-            .maxResults(10)
-            .searchTaskRequest(searchTaskRequest)
-            .roleAssignments(roleAssignmentsWithGrantTypeBasic(Classification.PRIVATE))
-            .expectedAmounfOfTasksInResponse(4)
-            .expectedTotalRecords(4)
-            .expectedTaskDetails(newArrayList(
-                                     "8d6cc5cf-c973-11eb-bdba-0242ac111007", "1623278362431007",
-                                     "8d6cc5cf-c973-11eb-bdba-0242ac111008", "1623278362431008",
-                                     "8d6cc5cf-c973-11eb-bdba-0242ac111027", "1623278362431027",
-                                     "8d6cc5cf-c973-11eb-bdba-0242ac111028", "1623278362431028"
-                                 )
-            ).build();
-
-        final TaskQueryScenario restrictedClassification = TaskQueryScenario.builder()
-            .scenarioName("basic_grant_type_with_classification_as_restricted")
-            .firstResult(0)
-            .maxResults(10)
-            .searchTaskRequest(searchTaskRequest)
-            .roleAssignments(roleAssignmentsWithGrantTypeBasic(Classification.RESTRICTED))
-            .expectedAmounfOfTasksInResponse(6)
-            .expectedTotalRecords(6)
-            .expectedTaskDetails(newArrayList(
-                                     "8d6cc5cf-c973-11eb-bdba-0242ac111007", "1623278362431007",
-                                     "8d6cc5cf-c973-11eb-bdba-0242ac111008", "1623278362431008",
-                                     "8d6cc5cf-c973-11eb-bdba-0242ac111009", "1623278362431009",
-                                     "8d6cc5cf-c973-11eb-bdba-0242ac111027", "1623278362431027",
-                                     "8d6cc5cf-c973-11eb-bdba-0242ac111028", "1623278362431028",
-                                     "8d6cc5cf-c973-11eb-bdba-0242ac111029", "1623278362431029"
-                                 )
-            ).build();
-
-        return Stream.of(
-            publicClassification,
-            privateClassification,
-            restrictedClassification
-        );
-    }
-
     private static Stream<TaskQueryScenario> grantTypeBasicErrorScenario() {
```

Comment:
Do we still need to test for this ?
.grantType(GrantType.BASIC)

Reply:
This time we are testing invalid scenarios with Basic Grant type. I think we should keep in case of we reintroduce Basic type in the future.